src/app/modules/common/layouts/admin-layout/admin-layout.component.scss

```diff
@@ -58,6 +59,13 @@
   top: 13px;
 }
 
+.logo:focus-visible,
+.logo-text:focus-visible {
+  background: rgba(120, 120, 120, 0.2) !important;
```

Comment:
Avoid using `!important` so much if possible. It can be used when nothing else works. Try to find out what's stopping the style from getting applied in the browser inspector maybe.